File: app/src/main/java/com/example/kineduexample/data/network/model/DataArticles.java
package com.example.kineduexample.data.network.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class DataArticles {
    @SerializedName("articles")
    List<Articles> articles;

    public List<Articles> getArticles() {
        return articles;
    }

    public void setArticles(List<Articles> articles) {
        this.articles = articles;
    }

    @Override
    public String toString() {
        return "DataArticles{" +
                "articles=" + articles +
                '}';
    }
}

File: app/src/main/java/com/example/kineduexample/data/network/model/DataArticles.kt
package com.example.kineduexample.data.network.model

import com.google.gson.annotations.SerializedName

class DataArticles {
    @SerializedName("articles")
    var articles: List<Articles>? = null

    override fun toString(): String {
        return "DataArticles{" +
                "articles=" + articles +
                '}'.toString()
    }
}
